pkg/process/v6/transformers/github/transform_test.go

```diff
@@ -640,8 +638,7 @@ func TestGetRanges(t *testing.T) {
 		ranges = append(ranges, rng...)
 	}
 
-	assert.Equal(t, 1, len(errors))
-	assert.ErrorIs(t, errors[0], version.ErrFallbackToFuzzy)
```

Comment:
This removal is a bit surprising. I think we may be undoing https://github.com/anchore/grype/pull/2651. (See also comment above on the implementation.)